### PR TITLE
Added order update. Added some specs

### PR DESCRIPTION
**Pull request by domenic99**
Order update

### CODE REVIEW

File: spec/controllers/orders_controller_spec.rb [hunk 1]
```diff
@@ -152,14 +188,14 @@
       it 'should return 422' do
```
Comment (rakvium) [inline spec/controllers/orders_controller_spec.rb]: @domenic99 oops, it actually checks for 403 :sweat_smile:

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -24,6 +24,15 @@ def show
   end
 
   def update
+    if @current_user.try(:instance_of?, Driver)
+      return render json: { 'error' => 'You are not allowed  to change an order' }, status: 401
```
Comment (rakvium) [inline app/controllers/orders_controller.rb]: @domenic99 I think that additional space is redundant here.

Comment (rakvium) [inline app/controllers/orders_controller.rb]: @domenic99 and it's better to use 403 if user is already logged in.

Comment (domenic99) [inline app/controllers/orders_controller.rb]: @rakvium But driver is not allowed to update order

Comment (rakvium) [inline app/controllers/orders_controller.rb]: @domenic99, yes, he isn't. But it's because he is driver, not because he is not logged in :wink:

**Pull request merged**